opencti-platform/opencti-front/src/components/dataGrid/DataTable.tsx

```diff
@@ -104,6 +107,17 @@ const DataTableInternalFilters = ({
           </Alert>
         </div>
       )}
+      {storageKey === 'restrictedEntities' && (
+        <div style={{ width: '100%', marginBottom: 20 }}>
+          <Alert
+            severity="info"
+            variant="outlined"
+            style={{ padding: '0px 10px 0px 10px' }}
+          >
+            {('This list displays all the entities that have some access restriction enabled, meaning that they are only accessible to some specific users. You can remove this access restriction on this screen.')}
```

Comment:
Looks like i18n is missing.
```suggestion
            {t_i18n('This list displays all the entities that have some access restriction enabled, meaning that they are only accessible to some specific users. You can remove this access restriction on this screen.')}
```